app/scenes/channel/channel.js

```diff
@@ -122,6 +130,14 @@ export default class Channel extends React.PureComponent {
                         />
                     </Drawer>
                 </ChannelDrawer>
+                <ChannelModal
+                    visible={this.state.channelDropdownOpen}
+                    theme={theme}
```

Comment:
Actually is being used [here](https://github.com/mattermost/mattermost-mobile/pull/162/files#diff-1804bb96f74188c7c1ada7d7ffb73bbfR44) so that's fine